lib/google/apis/options.rb

```diff
@@ -73,6 +74,8 @@ class RequestOptions
       #   @return [Boolean] True if response should be returned in raw form instead of deserialized.
       # @!attribute [rw] api_format_version
       #   @return [Fixnum] Version of the error format to request/expect.
+      # @!attribute [rw] opencensus_span_name
+      #   @return [String,nil] Name of OpenCensus span to generate, or nil to disable OpenCensus.
```

Comment:
Maybe good to mention the default value in the docs?

Reply:
Done.